portfolio/src/main/webapp/script.js

```diff
@@ -83,6 +83,7 @@ function getKdramaRecommendation() {
 function getComments(){
   fetch("/data").then(result => result.json()).then((comments) => {
     const section = document.getElementById("comments");
+    section.innerHTML = "";
```

Comment:
I'm not sure if this will work and you probably need to remove the child elements one by one, have you tried it locally? 🤔

Reply:
yup! i tried pressing the button multiple of times and it did not show any duplicate 👍

Reply:
Great!